recipe/meta.yaml

```diff
@@ -24,22 +24,22 @@ requirements:
     - flit-core >=3.2,<4
   run:
     - python
-    - libmambapy >=0.23
-    - conda >=4.13
-    - importlib_metadata
+    - libmambapy >=1.0
```

Comment:
I guess the upstream `libmambapy >=0.23` is outdated https://github.com/conda/conda-libmamba-solver/blob/990b25213b4f6c7a39c96533c928a8d690d12a86/pyproject.toml#L27

Reply:
https://github.com/conda/conda-libmamba-solver/pull/90

Reply:
I'll do a 22.11.1 with that